scripts/run-integration-test.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const { promises: fs } = require('fs');
+const { promisify } = require('util');
+const { once } = require('events');
+const proc = require('child_process');
+const execFile = promisify(proc.execFile);
+
+const [,, command = 'run', ...args] = process.argv;
+
+if (command === 'run') {
+  run(args);
+} else if (command === 'run-tarball') {
+  run(args.slice(1), args[0]);
+} else if (command === 'pack') {
+  pack(args[0]);
+}
```

Comment:
Should we add an `else` here with an error message? Otherwise running the script by itself will exit and do nothing.

Reply:
Note that there is a default set in line 10. It's not the most explicit piece of code. Should we change it so the default is more explicit?

Reply:
Ah, I had no idea that's what that assignment did, but that could also be my poor JavaScript skills :sweat_smile:

Reply:
There should still be an else anyway for unknown subcommands.